ollama.yaml

```diff
@@ -24,7 +24,7 @@ pipeline:
 
   - uses: go/bump
     with:
-      deps: golang.org/x/crypto@v0.17.0 google.golang.org/protobuf@v1.33.0 golang.org/x/net@v0.23.0
+      deps: golang.org/x/crypto@v0.17.0 google.golang.org/protobuf@v1.33.0 golang.org/x/net@v0.23.0 github.com/ollama/ollama@v0.1.29
```

Comment:
This fix seems so wrong !!!

Reply:
it is 
cc @hectorj2f

Reply:
it is a main module verification. Let me change the repository.